packages/raylib/raylib.0.3/opam

```diff
@@ -0,0 +1,54 @@
+opam-version: "2.0"
+synopsis: "OCaml bindings for raylib"
+description: "OCaml bindings for raylib"
+maintainer: ["tobiasjammer@gmail.com"]
+authors: ["Tobias Mock"]
+license: "MIT"
+homepage: "https://github.com/tjammer/raylib-ocaml"
+bug-reports: "https://github.com/tjammer/raylib-ocaml/issues"
+depends: [
+  "dune" {>= "2.8"}
+  "dune-configurator"
+  "ctypes"
+  "ppx_cstubs"
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/tjammer/raylib-ocaml.git"
+depexts: [
+  ["mesa-common-dev" "libx11-dev" "libxrandr-dev" "libxi-dev" "xorg-dev" "libgl1-mesa-dev" "libglu1-mesa-dev"] {os-family = "debian"}
+  ["mesa-libGL-devel" "libX11-devel" "libXrandr-devel" "libXi-devel" "libXcursor-devel" "libXinerama-devel"] {os-family = "fedora"}
+  ["mesa-libGL-devel" "libX11-devel" "libXrandr-devel" "libXi-devel" "libXcursor-devel" "libXinerama-devel"] {os-family = "rhel"}
+  ["mesa-dev" "libx11-dev" "libxrandr-dev" "libxi-dev" "libxcursor-dev" "libxinerama-dev"] {os-distribution = "alpine"}
+  ["Mesa-libGL-devel" "libX11-devel" "libXrandr-devel" "libXi-devel" "libXcursor-devel" "libXinerama-devel"] {os-family = "suse"}
+  ["system:libX11-devel" "system:libXrandr-devel" "system:libXi-devel" "system:libXcursor-devel" "system:libXinerama-devel"] {os-distribution = "cygwinports"}
+  ["mesa-libs" "libX11" "libXrandr" "libXi" "libXcursor" "libXinerama"] {os-distribution = "freebsd"}
+  ["libxcursor" "libxrandr" "libxinerama" "libxi" "mesa"] {os-family = "arch"}
```

Comment:
is there any chance you could split these dependencies into their own `conf-*` packages (see `conf-libX11` for example) ? I know this is a lot but it would help you and us maintain this package and others in the long term

Reply:
Yes, this sounds reasonable, will do.